src/styles/saas-list.less

```diff
@@ -49,10 +48,10 @@
       color: @saas-text;
       display: flex;
       flex: 1 1 0%; // IE 11
+      padding: (@grid-gutter-width / 2);
```

Comment:
#142 also makes this change, but adds the removal of line 24.  Maybe add the removal of line 24 and we can close #142?